test/setup/platforms/openshift4.ts

```diff
@@ -72,22 +72,25 @@ async function tryDescribingResourceToFile(
   name: string,
   namespace: string,
 ): Promise<void> {
-  return kubectl
-    .describeKubernetesResource(kind, name, namespace)
-    .then((description) => {
-      const fileName = `${kind}-${name}-${namespace}`;
-      const filePath = process.env.CI
-        ? // The directory is generated by CircleCI config (see .circleci/config.yml).
-          `/tmp/logs/test/integration/openshift4/${fileName}`
-        : `${tmpdir()}/${fileName}`;
-      writeFileSync(filePath, description);
-      console.log(`Description for ${kind} ${name} is stored in ${filePath}`);
-    })
-    .catch(() =>
-      console.log(
-        `Could not describe ${kind} ${name} in namespace ${namespace}`,
-      ),
+  try {
+    const description = await kubectl.describeKubernetesResource(
```

Comment:
Do you need to `return` this?

Reply:
the function is now `await`ed and the work done previously in the `.then` is now done in L81-87

also the return type is `Promise<void>` :)

Reply:
Never mind,  you don't.